src/Microsoft.Identity.Web/Base64UrlHelpers.cs

```diff
@@ -7,80 +7,80 @@
 
 namespace Microsoft.Identity.Web
 {
-    internal static class Base64UrlHelpers
-    {
-        private const char Base64PadCharacter = '=';
-        private const char Base64Character62 = '+';
-        private const char Base64Character63 = '/';
-        private const char Base64UrlCharacter62 = '-';
-        private const char Base64UrlCharacter63 = '_';
-        private static readonly Encoding TextEncoding = Encoding.UTF8;
+	internal static class Base64UrlHelpers
+	{
+		private const char Base64PadCharacter = '=';
+		private const char Base64Character62 = '+';
+		private const char Base64Character63 = '/';
+		private const char Base64UrlCharacter62 = '-';
+		private const char Base64UrlCharacter63 = '_';
+		private static readonly Encoding TextEncoding = Encoding.UTF8;
 
-        private static readonly string DoubleBase64PadCharacter = string.Format(CultureInfo.InvariantCulture, "{0}{0}", Base64PadCharacter);
+		private static readonly string DoubleBase64PadCharacter = string.Format(CultureInfo.InvariantCulture, "{0}{0}", Base64PadCharacter);
```

Comment:
This is a good example of where string.Format isn't needed
```suggestion
		private const string DoubleBase64PadCharacter = Base64PadCharacter + Base64PadCharacter;
```